test/cpp/end2end/thread_stress_test.cc

```diff
@@ -225,8 +225,9 @@ class CommonStressTestAsyncServer : public BaseClass {
 
     void* ignored_tag;
     bool ignored_ok;
-    while (cq_->Next(&ignored_tag, &ignored_ok))
+    while (cq_->Next(&ignored_tag, &ignored_ok)) {
       ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done